packages/fxa-support-panel/src/remote-lookup/remote-responses.dto.ts

```diff
@@ -17,7 +22,15 @@ interface Subscription {
   subscription_id: string;
 }
 
-export interface SubscriptionResponse extends Array<Subscription> {}
+type PlaySubscription = Omit<AbbrevPlayPurchase, 'expiry_time_millis'> & {
+  product_id: string;
+  expiry: string;
```

Comment:
Why is this named `expiry` instead? I don't see a similar key e.g. on `WebSubscription` above. I guess because it's not a ms value anymore but a formatted Date string?